create-user-from-guest-order.php

```diff
@@ -45,10 +49,88 @@ public function run()
 
         // Create user from guest order
         add_action('woocommerce_new_order', array($this, 'createUserFromGuestOrder'), 10, 1);
+
         // Create user from guest order when admin update the order
         add_action('woocommerce_process_shop_order_meta', array($this, 'createUserFromGuestOrder'), 999, 1);
+
+        // Maybe show a button to create an user.
+        add_action( 'woocommerce_admin_order_data_after_order_details', array( $this, 'maybe_show_create_user_button' ) );
+
+        // AJAX processing for user creation.
+        add_action( 'wp_ajax_cufgo_maybe_create_user', array( $this, 'maybe_create_user') );
+
+        // easy link to our settings page
+        add_filter( 'plugin_action_links_create-user-from-guest-order-wp-plugin/create-user-from-guest-order.php', array( $this, 'plugin_action_links' ) );
+
+    }
+
+    /**
+     * Add a link to our settings page at plugins listing page.
+     * @param  array $links
+     * @return array
+     */
+    public function plugin_action_links( $links ) {
+
+        $plugin_links = array( '<a href="' .  admin_url( 'admin.php?page=wc-settings' ) . '">' . esc_html__( 'Settings', 'create-user-from-guest-order' ) . '</a>' );
+
+        return array_merge( $plugin_links, $links );
+    }
+
+
+    /**
+     * Show create user button if order customer does not exist.
+     *
+     * @param WC_Order $order
+     */
+    public static function maybe_show_create_user_button( $order ) {

```

Comment:
@greguly As this function is not used in more than 1 place, please consider not using static method

Reply:
@s-azizkhan Just fixed.

Reply:
@greguly I guess you missed removing the static method

Reply:
Oh dear, I removed but missed to commit.

Reply:
@s-azizkhan Done!